e2e/.#rdctl.e2e.spec.ts

```diff
@@ -0,0 +1 @@
+ericp@chirico.local.14112
```

Comment:
This needs to be removed too — might want to edit your `.git/info/exclude` (or `~/.config/git/exclude` or whatever) to drop `.#`

Reply:
Yup, updated `~/.gitignore`